pkgs/tools/networking/ogen/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  pkgs,
+  lib,
+}:
+pkgs.buildGoModule rec {
+  pname = "ogen";
+  version = "1.0.0";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "ogen-go";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-wIcxICfBY5YHVqKHtHd2wzcaDmkzgp7eD6zEvnsRaIU=";
+  };
+
+  vendorHash = "sha256-EL8FcAnDMekHBIDRdGqQO4JNESjGC7MQ5aUcDkRqrbg=";
+  doCheck = false;
```

Comment:
Looks like the package builds fine even with checks enabled, so this could be removed.